chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx

```diff
@@ -33,14 +33,15 @@ export const LoadingPaneStateful = (props: ILoadingPaneProps) => {
     };
 
     const errorUIControlProps: ILoadingPaneControlProps = {
+        ...props.controlProps,
         id: "oc-lcw-alert-pane",
         dir: state.domainStates.globalDir,
         titleText: "Chat is failing to load.",
         subtitleText: "Please Close the chat and try again.",
```

Comment:
nit : Can we make the c as small in subtitle text?
"Please close the chat and try again"